src/vs/workbench/contrib/terminal/browser/terminalProcessManager.ts

```diff
@@ -189,7 +189,13 @@ export class TerminalProcessManager extends Disposable implements ITerminalProce
 				// Flow control is not needed for ptys hosted in the same process (ie. the electron
 				// renderer).
 				if (shellLaunchConfig.attachPersistentTerminal) {
-					this._process = await this._terminalInstanceService.attachToProcess(shellLaunchConfig.attachPersistentTerminal.id);
```

Comment:
we don't need this code right now to fix the issue, but may want it as a safeguard?

Reply:
Yeah this looks good